ptrack/templatetags/ptrack.py

```diff
@@ -18,7 +16,7 @@
 def ptrack(*args, **kwargs):
     """Generate a tracking pixel html img element."""
     if settings.PTRACK_APP_URL:
-        encoded_dict = {'ptrack_encoded_data': ptrack_encoder.encrypt(*args, **kwargs)}
+        encoded_dict = {'ptrack_encoded_data': ptrack_encoder.encrypt_string(*args, **kwargs)}
```

Comment:
Without the switch from bytestring to string, ptrack was producing malformed urls because [Django 2.0 dropped support for bytestrings in `reverse`](https://docs.djangoproject.com/en/2.2/releases/2.0/#removed-support-for-bytestrings-in-some-places)